go.mod

```diff
@@ -0,0 +1,10 @@
+module health-check
+
+require (
+	github.com/mitchellh/go-homedir v1.1.0
+	github.com/oliveagle/jsonpath v0.0.0-20180606110733-2e52cf6e6852
+	github.com/spf13/cobra v0.0.5
+	github.com/spf13/viper v1.6.1
```

Comment:
All these appear to be new dependencies -- any approval needed to introduce them?

Reply:
Good point, who do I reach out to for approval?

Reply:
I'd ask @michaelestrin.  I think you might also need an Attributions.txt and although the license on these components look fine, you should check their dependencies to make sure that all the sublicenses are compatible.

Reply:
Redirecting this to @tsconn23 for clarification/guidelines on how to add additional external dependencies.

Related - https://github.com/edgexfoundry/go-mod-messaging/issues/32

Reply:
I added an `Attribution.txt` to give credit to the referenced open-sourced projects